### PR TITLE
Modify the opacity of unsupported add-ons (fix #2289)

### LINKED ISSUE #2289: Always show if addon can be installed in listings/seach results
[Example search](https://addons.mozilla.org/en-us/firefox/search/?q=minimize+to+tray&status=4&sourceid=Mozilla-search)

Always show if an addon can be installed in listings/seach results so one can easily view of a search yielded something useful for not or which addon can be installed.

### PR DESCRIPTION
**Pull request by tofumatt**
### Before

screenshot 2016-04-13 19 44 19

### After

screenshot 2016-04-13 19 26 51
screenshot 2016-04-13 19 26 47

### CODE REVIEW

File: static/js/zamboni/buttons.js [hunk 1]
```diff
@@ -274,6 +274,7 @@ var installButton = function() {
             warn(gettext('Not available for your platform'));
             $button.addClass('concealed');
             $button.first().css('display', 'inherit');
+            $button.closest('.item.addon').addClass('incompatible');
```
Comment (muffinresearch) [inline static/js/zamboni/buttons.js]: How was the CSS keyed before? Is there not a way to change this without modifying the JS?

Comment (tofumatt) [inline static/js/zamboni/buttons.js]: It's always been done with JS, the add-on is output with just the classes `item` and `addon` before this.

Comment (tofumatt) [inline static/js/zamboni/buttons.js]: I'm assuming we modified some CSS that prevented it doing something directly to the CSS in JS. But I'm honestly not clear on that.

**Comment by muffinresearch** [comment]
r+

**Comment by muffinresearch** [comment]
This will need some extra QA just to check in-case of unexpected side-effects - I'd suggest adding a comment to the bug once this lands.

**Pull request merged**